package.json

```diff
@@ -43,6 +43,7 @@
     "react-native": "*"
   },
   "devDependencies": {
+    "@types/react-native": "^0.71.4",
```

Comment:
I think it's better same with react-native.

```suggestion
    "@types/react-native": "^0.59.10",
```

Reply:
I checked the versions of "@types/react-native" but it doesn't have the version "0.59.10".

![image](https://user-images.githubusercontent.com/40392351/226916483-84f1466f-1a36-47f1-8fc5-dbf95e42ad5a.png)

The suggestion is to update react-native to version "0.71.4" as well.

Reply:
OK. It's better to choose "[0.60.31](https://www.npmjs.com/package/@types/react-native/v/0.60.31)"

Reply:
Version updated.